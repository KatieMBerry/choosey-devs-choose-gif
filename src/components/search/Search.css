.Search {
    display: block;
    margin: auto;
    padding: 1rem;
}
