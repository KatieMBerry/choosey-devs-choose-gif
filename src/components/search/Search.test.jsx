import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Search from './Search';

describe('Search component', () => {
    afterEach(() => cleanup());
    it('renders a Search input', () => {
        const { asFragment } = render(<Search
            search='my_search_query'
            onChange={jest.fn()}
        />);
        expect(asFragment()).toMatchSnapshot();
    });
});
