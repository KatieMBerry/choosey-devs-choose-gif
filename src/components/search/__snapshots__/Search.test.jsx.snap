// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders a Search input 1`] = `
<DocumentFragment>
  <input
    class="Search"
    placeholder="Search here..."
    type="text"
    value="my_search_query"
  />
</DocumentFragment>
`;
